src/DssSpell.t.sol

```diff
@@ -2312,103 +2295,12 @@ contract DssSpellTest is DSTest, DSMath {
-        assertEq(join.wards(address(esm)), 1);          // Required in case of gov. attack
-        assertEq(join.wards(address(directMom)), 1);    // Zero-delay shutdown for Aave gov. attack
-
-        // Check the bar/tau/king are set correctly
-        assertEq(join.bar(), bar);
-        assertEq(join.tau(), tau);
-        assertEq(join.king(), pauseProxy);
-
-        // Set the target bar to be super low to max out the debt ceiling
-        giveAuth(address(join), address(this));
-        join.file("bar", 1 * RAY / 10000);     // 0.01%
-        join.deny(address(this));
-        join.exec();
-
-        // Module should be maxed out
-        (,,, uint256 line,) = vat.ilks(_ilk);
-        (uint256 ink, uint256 art) = vat.urns(_ilk, address(join));
-        assertEq(ink*RAY, line);
-        assertEq(art*RAY, line);
-        assertGe(token.balanceOf(address(join)), ink - 1);         // Allow for small rounding error
-
-        // Disable the module
-        giveAuth(address(join), address(this));
-        join.file("bar", 0);
-        join.deny(address(this));
-        join.exec();
-
-        // Module should clear out
-        (ink, art) = vat.urns(_ilk, address(join));
-        assertLe(ink, 1);
-        assertLe(art, 1);
-        assertEq(token.balanceOf(address(join)), 0);
-    }
-
     function testCollateralIntegrations() public {
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new collateral tests here
-        checkDirectIlkIntegration(
-            "DIRECT-AAVEV2-DAI",
-            DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI")),
-            ClipAbstract(addr.addr("MCD_CLIP_DIRECT_AAVEV2_DAI")),
-            addr.addr("PIP_ADAI"),
-            4 * RAY / 100,
-            7 days
-        );
-    }
-
-    function testDirectMom() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        assertEq(directMom.authority(), address(chief));
-
-        DirectDepositLike join = DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI"));
-
-        // Wind up the D3M
-        giveAuth(address(join), address(this));
-        join.file("bar", 1 * RAY / 10000);     // 0.01%
-        join.deny(address(this));
-        join.exec();
-        (, uint256 art) = vat.urns("DIRECT-AAVEV2-DAI", address(join));
-        assertGt(art, 0);
-
-        // Verify we can disable the D3M through the mom
-        hevm.store(
-            address(directMom),
-            0,
-            bytes32(uint256(address(this)))
-        );      // Set this contract as the owner
-        directMom.disable(address(join));
-        assertEq(join.bar(), 0);
-
-        // Should close out the D3M
-        join.exec();
-        (, art) = vat.urns("DIRECT-AAVEV2-DAI", address(join));
-        assertLe(art, 1);
     }
 
     function getExtcodesize(address target) public view returns (uint256 exsize) {
```

Comment:
`testNewChainlogValues` and `testNewIlkRegistryValues` can be cleanup up and commented, as well as `testCollateralIntegrations` to reduce testing time.